tests/testthat/test_regr_ranger.R

```diff
@@ -66,3 +66,17 @@ test_that("default_values", {
   values = default_values(learner, search_space, task)
   expect_names(names(values), permutation.of =  c("replace", "sample.fraction", "num.trees", "mtry.ratio"))
 })
+
+test_that("quantile prediction", {
+  learner = mlr3::lrn("regr.ranger", num.trees = 100, predict_type = "quantile")
+  learner$quantiles = c(0.1, 0.5, 0.9)
+  learner$quantile_response = 0.5
+  task = tsk("mtcars")
+
+  learner$train(task)
+  pred = learner$predict(task)
+  expect_matrix(pred$quantile)
+  tab = as.data.table(pred)
+  expect_names(names(tab), must.include = c("q0.1", "q0.5", "q0.9", "response"))
```

Comment:
Can I also get the mean and median?

Reply:
The order here is not arbitrary, the quantiles should be sorted in increasing order.

Reply:
Bernd says no

Reply:
🤷‍♂️